### DISCUSSION TITLE
算法学习 - 计算机学习指南

### DISCUSSION THREAD

**Discussion by giscus** [meowrain/doc-for-sxau#44] [Ideas]
# 算法学习 - 计算机学习指南

计算机学习指南，这里有从网络上收集的计算机各种方面的知识技能，可供查阅和学习

https://cs.meowrain.cn/%E5%9F%BA%E6%9C%AC%E5%8A%9F/%E6%95%B0%E6%8D%AE%E7%BB%93%E6%9E%84/algorithm-learn/?h=%E7%AE%97%E6%B3%95

**Comment by meowrain** [comment]
https://oi-wiki.org/math/
